.github/workflows/scripts/wait-for-image.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+#
+# Wait for an image to appear on Quay.io
+#
+set -euo pipefail
+
+NAME="$1"
+TAG="$2"
+
+check_not_empty \
+    NAME \
+    TAG \
+    \
+    QUAY_TOKEN \
+    DRY_RUN
+
+find_tag() {
+    curl --silent --show-error --fail --location \
+        -H "Authorization: Bearer $QUAY_TOKEN" \
+        -X GET "https://quay.io/api/v1/repository/rhacs-eng/$1/tag?specificTag=$2" |
+        jq -r ".tags[0].name"
+}
+
+# Seconds:
+TIME_LIMIT=1200
+INTERVAL=10
```

Comment:
I think CircleCI used to wait up to 30 minutes, 20 mins is also fine.
I hope we will not run into rate-limiting - maybe interval 30 would be slightly better?